components/eam/tools/topo_tool/bin_to_cube/bin_to_cube.F90

```diff
@@ -254,9 +268,29 @@ program convterr
       idx(i,j) = icube
       idy(i,j) = jcube
       idp(i,j) = ipanel
+  !WRITE(*,*) "terr_cube(icube,jcube,ipanel) at j,i = ",icube,jcube,ipanel,j,i,terr_cube(icube,jcube,ipanel)
     END DO
   END DO
+  
+
 
+  DO k=1,6
+    DO j=1,ncube          
+      DO i=1,ncube
+        IF (ABS(weight(i,j,k))<1.0E-9) THEN
+          WRITE(*,*) "there is no lat-lon grid point in cubed sphere cell ",i,j,k
+          WRITE(*,*) "fatal error"
+          write(*,*) "weight     ",i,j,k,weight(i,j,k)
+          STOP
+        ELSE
+          terr_cube        (i,j,k) = terr_cube        (i,j,k)/weight(i,j,k)                
+!+++ARH
+          landfrac_cube    (i,j,k) = landfrac_cube    (i,j,k)/weight(i,j,k)                
```

Comment:
Do we need to keep landfrac around?

Reply:
in the updates, the second commit removed all landfrac related content

for my own experience so far landfrac is not needed so i vote to remove it, but it can be reverted if others want it?

Reply:
let's drop it and then add it back if something breaks.